### ISSUE TITLE
Abnormality of heart axis

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6606]
**Preferred term label:** Abnormality of heart axis

**Synonyms** Abnormality of cardiac axis

**Definition (free text, please give PubMed ID)**
The cardiac axis gives an idea of the overall direction of electrical activity. In healthy individuals, the axis lies between -30° and +90º.

**Parent term (use hpo.jax.org/app)**
Abnormal EKG HP:0003115

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
I think we mean the ventricular (QRS) axis here. I will label the term
Abnormal ventricular axis
syn Abnormal QRS axis
def: Any deviation from the normal direction of the ventricular axis. The left ventricle makes up most of the heart muscle under normal circumstances and therefore generates the most electrical force visible on the EKG. The normal ventricular axis is directed downward and slightly towards the left. The ventricular axis can be determined by analyzing the QRS complex, which represents ventricular depolarization. 
Comment: The normal adult QRS axis is between -30 degrees and +90 degrees, which is directed downward and to the left.
